.github/workflows/e2e-tests.yml

```diff
@@ -1,5 +1,7 @@
 name: e2e
 on:
+  schedule:
+    - cron: '30 5,17 * * *' # run this every day at 5:30 and 17:30 UTC (00:30 and 12:30 ET)
```

Comment:
How do you feel about `0 */12 * * *` for readability?

Reply:
I was hoping to keep things in ET. I like getting a run in over lunch.

Reply:
Yeah that's reasonable to me. We may want to re-evaluate the twice-a-day frequency at some point if it's too costly over the long term, but likely fine in the short term to help reduce the overall CI noise and create attempt to create metrics over time.